scripts/update_avalanchego_version.sh

```diff
@@ -7,6 +7,11 @@ if ! [[ "$0" =~ scripts/update_avalanchego_version.sh ]]; then
   exit 255
 fi
 
+SUBNET_EVM_PATH=$(
```

Comment:
Looks like this is unused currently, did you mean to use it to run `scripts/constants.sh`?

Reply:
yes, updated